css/style.css

```diff
@@ -149,24 +142,15 @@ img {
 #info h2 {
     margin: 17px 0 15px 0;
     border: 0;
-    font-family: 'Georgia', 'Ovo', serif;
     color: #2b7dbc;
-    font-size: 1.75em;
+    font-size: 2.2rem;
     font-weight: normal;
 }
 
-#info p {
-  line-height: 1.5;
-}
-
 #content {
-    text-align: justify;
```

Comment:
Why the alignment change?

Reply:
I feel like text-align: justify messes up the word spacing in paragraphs and only makes the text harder to read IMO. Has no benefit over normal text alignment (left).